common/jersey2-client/src/main/java/com/bazaarvoice/emodb/common/jersey2/BaseRetryPolicy.java

```diff
@@ -0,0 +1,143 @@
+package com.bazaarvoice.emodb.common.jersey2;
+
+import com.bazaarvoice.emodb.client2.EmoClientException;
+import com.bazaarvoice.emodb.client2.EmoResponse;
+import com.bazaarvoice.emodb.common.json.JsonStreamProcessingException;
+import com.bazaarvoice.emodb.common.json.JsonStreamingEOFException;
+import com.bazaarvoice.emodb.uac.api.EmoApiKeyExistsException;
+import com.bazaarvoice.emodb.uac.api.EmoApiKeyNotFoundException;
+import com.bazaarvoice.emodb.uac.api.EmoRoleExistsException;
+import com.bazaarvoice.emodb.uac.api.EmoRoleNotFoundException;
+import com.bazaarvoice.emodb.uac.api.InsufficientRolePermissionException;
+import com.bazaarvoice.emodb.uac.api.InvalidEmoPermissionException;
+import com.google.common.base.Predicates;
+import com.google.common.base.Throwables;
+import dev.failsafe.RetryPolicy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.bazaarvoice.emodb.common.api.UnauthorizedException;;
+
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import java.time.Duration;
+import com.bazaarvoice.emodb.common.api.ServiceUnavailableException;
+
+public class BaseRetryPolicy {
```

Comment:
would create a proper utility class
```suggestion
public final class RetryPolicy {

    private RetryPolicy() {
    }
```